docs/spark-queries.md

```diff
@@ -227,6 +227,20 @@ SELECT * FROM prod.db.table.history
 **This shows a commit that was rolled back.** The example has two snapshots with the same parent, and one is *not* an ancestor of the current table state.
 {{< /hint >}}
 
+### Metadata Logs
+
+To show table metadata logs:
+
+```sql
+SELECT * from prod.db.table.metadata_logs
+```
+
+| timestamp_millis | file | latest_snapshot_id | latest_schema_id | latest_sequence_number |
+| -- | -- | -- | -- | -- |
+| 1658211678838 | s3://.../table/metadata/00000-9441e604-b3c2-498a-a45a-6320e8ab9006.metadata.json | null | null | null |
```

Comment:
I think we should convert this to a timestamp rather than displaying millis. That's what we do in other tables.

Reply:
makes sense, done !